libs/test-cli/Cargo.toml

```diff
@@ -9,8 +9,8 @@ colored = "2"
 structopt = "0.3.8"
 enumflags2 = "0.7"
 dmmf = { path = "../../query-engine/dmmf" }
-migration-core = { path = "../../migration-engine/core" }
```

Comment:
For context, as I'm not super aware of what this looked like prior, whilst we still had an official introspection engine: Did we have an introspection-core, and was that then merged into migration-core too? Or does the new schema-core currently only handle migrations

Reply:
We had introspection-core before, the schema-core handles both.